lib/config.dart

```diff
@@ -103,6 +104,19 @@ class Config {
         (cache.binPath == null || !io.File(cache.binPath).existsSync()))
       await _askForBinPath();
 
+    /** Generate Discord Id's */
+    if (cache.ids.length != userNumber) {
+      if (userNumber > cache.ids.length) {
+        // More Id's (add)
+        int newIds = userNumber - cache.ids.length;
+        for (int i = 0; i < newIds; i++) cache.ids.add(Uuid().v4());
+      } else if (userNumber < cache.ids.length) {
+        // Less Id's (fresh list)
+        for (int i = 0; i < userNumber; i++) cache.ids.add(Uuid().v4());
+      }
+      cache.save();
+    }
```

Comment:
How we do this is up to debate, re-using most of your code, this is what came to mind.

Reply:
Nice! I didn't think about accessing cache.ids from config.
Decreasing userNumber is something I overlooked. Thanks!